front/app/containers/Admin/reporting/containers/ReportBuilder/index.tsx

```diff
@@ -137,6 +143,14 @@ const ReportBuilderWrapper = () => {
   const { data: report } = useReport(reportId);
   const { data: reportLayout } = useReportLayout(reportId);
 
+  const [search] = useSearchParams();
+  const [templateProjectId] = useState(search.get('templateProjectId'));
+  const [templatePhaseId] = useState(search.get('templatePhaseId'));
+
+  useEffect(() => {
+    removeSearchParams(['templateProjectId', 'templatePhaseId']);
```

Comment:
Removing these parameters immediately to avoid confusion, in particular people reloading the page when they haven't saved and thinking it's saved because the template is just re-applied